dhis-2/dhis-api/src/main/java/org/hisp/dhis/relationship/RelationshipType.java

```diff
@@ -52,6 +52,12 @@
 
     private String description;
 
+    private boolean bidirectional = false;
+
+    private String fromToName;

```

Comment:
What is the name of a uni-directional relationship?

Reply:
answering my own question: fromToName

Reply:
fromToName would be the same as "name" is now.